contracts/tasks/deploy/maci/02-gatekeepers.ts

```diff
@@ -72,5 +76,39 @@ deployment.deployTask("full:deploy-gatekeepers", "Deploy gatekeepers").then((tas
         network: hre.network.name,
       });
     }
+
+    const isSupportedGitcoinGatekeeperNetwork = ![
+      ESupportedChains.Hardhat,
+      ESupportedChains.Coverage,
+      ESupportedChains.Sepolia,
```

Comment:
Why sepolia is not supported?

Reply:
https://docs.passport.xyz/building-with-passport/smart-contracts/contract-reference#decoder-contract-addresses 
They only have it on op-sepolia and linea